connector/docker-integration-tests/src/test/scala/org/apache/spark/sql/jdbc/v2/V2JDBCTest.scala

```diff
@@ -48,12 +48,26 @@ private[v2] trait V2JDBCTest extends SharedSparkSession with DockerIntegrationFu
 
   def notSupportsTableComment: Boolean = false
 
-  def defaultMetadata(dataType: DataType = StringType): Metadata = new MetadataBuilder()
+  def defaultMetadata(
+      dataType: DataType = StringType,
+      remoteTypeName: String = "STRING"): Metadata = new MetadataBuilder()
     .putLong("scale", 0)
     .putBoolean("isTimestampNTZ", false)
     .putBoolean("isSigned", dataType.isInstanceOf[NumericType])
+    .putString("remoteTypeName", remoteTypeName)
     .build()
 
+  def removeMetadataFromAllFields(structType: StructType, metadataKey: String): StructType = {
```

Comment:
nit: maybe
```suggestion
  def getSchemaWithoutFieldsMetadata(structType: StructType, metadataKey: String): StructType = {
```
or just add doc to describe what function does, I think current name is also ok in that case.

Reply:
Added doc